package.json

```diff
@@ -15,12 +15,12 @@
   "dependencies": {
     "@brown-ccv/behavioral-task-trials": "^1.0.0",
     "@fortawesome/fontawesome-free": "^5.9.0",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^5.2.0-beta1",
```

Comment:
It's awkward to depend on a beta, but I wanted to see a clean build!  I am hoping we can just upgrade to 5.2.0 when it becomes GA / non-beta.